.github/workflows/vast.yaml

```diff
@@ -299,7 +299,7 @@ jobs:
           cmake -B "$BUILD_DIR" \
             -DCMAKE_INSTALL_PREFIX:STRING="${PWD}/opt/vast" \
             -DCPACK_PACKAGE_FILE_NAME:STRING="$PACKAGE_NAME" \
-            -DCPACK_GENERATOR:STRING=TGZ \
+            -DCPACK_GENERATOR:STRING=TGZ;DEB \
```

Comment:
The semicolon requires you to add quotes, otherwise bash does the wrong thing here.